### ISSUE TITLE
Workaround: Go get pulls old version

### ISSUE THREAD

**Issue by Forrin** [in-toto/attestation#307]
`go get github.com/in-toto/attestation` will pull version v0.1.0 instead of v1.0.0

```
$ go get github.com/in-toto/attestation
go: downloading github.com/in-toto/attestation v0.1.0
go: added github.com/in-toto/attestation v0.1.0
```

The issue, I believe, is that the tag for version v1.0.0 is v1.0 instead. If that tag can be updated to v1.0.0, then this issue should resolve. Until then..

The work around for now is to pull by Git SHA. `go get github.com/in-toto/attestation@ee16c68a11dfcfbdc891600cacd767896fe6e724`.

Or you can pull by branch name, such as main, `go get github.com/in-toto/attestation@main`

```
$ go get github.com/in-toto/attestation@ee16c68a11dfcfbdc891600cacd767896fe6e724
go: downloading github.com/in-toto/attestation v0.1.1-0.20230322150215-ee16c68a11df
go: upgraded github.com/in-toto/attestation v0.1.0 => v0.1.1-0.20230322150215-ee16c68a11df
```

If you try to pull by version tag of v1.0, then you get

```
$ go get github.com/in-toto/attestation@v1.0
go: github.com/in-toto/attestation@v1.0: no matching versions for query "v1.0"
```

**Comment by marcelamelara** [comment]
Thanks for this this issue @Forrin ! Others have recently reported this issue as well. We believe that cutting a new release (which has been in the works anyway) makes more sense at this point than renaming an old tag. We'll make sure to match the tag and release version this time.

**Comment by marcelamelara** [comment]
This should be resolved with the new v1.0.1 patch release.
